Gemfile

```diff
@@ -34,6 +34,11 @@ gem "pg"
 
 gem "devise"
 
+gem "faye-websocket", "0.10.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols :construction_worker_man: